main.js

```diff
@@ -0,0 +1,160 @@
+const list = document.querySelector('.list');
+const refresh = document.getElementById('delete-to-do');
+const addButton = document.getElementById('add');
+const clearAll = document.getElementById('clearall');
+
+const tasks = JSON.parse(localStorage.getItem('tasksList')) || [];
+
+function checkedTasksEvent(tasksArr, checkbox) {
+  checkbox.forEach((box) => box.addEventListener('change', (e) => {
+    const tmp = tasksArr.findIndex((el) => el.index === parseInt(e.target.id, 10));
+
+    if (e.target.checked === true) {
+      tasksArr[tmp].completed = true;
+      e.target.nextElementSibling.classList.add('completed');
+    } else {
+      tasksArr[tmp].completed = false;
+      e.target.nextElementSibling.classList.remove('completed');
+    }
+
+    JSON.parse(localStorage.getItem('tasksList'));
+    localStorage.setItem('tasksList', JSON.stringify(tasksArr));
+  }));
+}
+
+
+function addTaskToStorage() {
+  const tasksInfo = JSON.stringify(tasks);
+  localStorage.setItem('tasksList', tasksInfo);
+}
+
+function AddTask() {
+  const task = {
+    description: document.querySelector('.input-task').value,
+    completed: false,
+    index: tasks.length,
+  };
```

Comment:
- [ ]  Kindly change all regular functions to es6 arrow function syntax following the JavaScript best practices guide.